modern-redux-with-redux-toolkit-rtk-and-typescript-64f243c8/notes/05_react-creating-a-rootstate-type-and-typed-hooks-for-type-aware-redux-interactions.md

```diff
@@ -4,7 +4,7 @@
 
 <TimeStamp start="0:02" end="0:18">
 
-More on [dispatch](https://redux.js.org/usage/usage-with-typescript#define-root-state-and-dispatch-types)
+[dispatch](https://redux.js.org/tutorials/fundamentals/part-2-concepts-data-flow#dispatch) is how you update state in your store. You'll pass dispatch an action object.
```

Comment:
another example of adding more context, think progressive summarization. Why is the link useful?